src/templates/page-donation-aiusa.html

```diff
@@ -1491,13 +1521,64 @@ <h2 class="section-header">Payment Info</h2>
                   loading="lazy"
                   src="https://aaf1a18515da0e792f78-c27fdabe952dfc357fe25ebf5c8897ee.ssl.cf5.rackcdn.com/1839/verisign-2021.png?v=1613334465000"
                   data-engrid-data-src-processed="true"
+                  width="174"
+                  height="32"
+                  style="min-height: 38px"
                 />
               </div>
             </div>
           </div>
           <div
             class="en__component en__component en__component--column en__component--advcolumn body-footer"
-          ></div>
+          >
+          <div class="en__component en__component--contactblock" style="">
```

Comment:
Should we create a separated file for this? It does not look like this is related to donations.